clients/src/main/java/org/apache/kafka/clients/ClientUtils.java

```diff
@@ -111,6 +111,10 @@ public static ChannelBuilder createChannelBuilder(AbstractConfig config, Time ti
 
```

Comment:
nit: you'll need to follow the pr format for kafka repos, similar to https://github.com/linkedin/kafka/pull/329

Reply:
Thanks, modified the format. Lemme know if further adjustment is needed.